node/http.go

```diff
@@ -112,6 +113,75 @@ func makeRPCOverWebsocket(host string, port uint16, servers map[string]*jsonrpc.
 	return makeHTTPService(host, port, cors.Default().Handler(mux))
 }
 
+type ipcService struct {
```

Comment:
Can we move the ipc things to `node/ipc.go`? They're technically not http-related.

Reply:
Websocket and grpc is being kept there too. Maybe `node/ipc.go` > `node/rpc.go` is better solution? It would align with the way they are instantiated.  
```go
if cfg.HTTP {
	services = append(services, makeRPCOverHTTP(cfg.HTTPHost, cfg.HTTPPort, rpcServers, log, cfg.Metrics))
}
if cfg.Websocket {
	services = append(services, makeRPCOverWebsocket(cfg.WebsocketHost, cfg.WebsocketPort, rpcServers, log, cfg.Metrics))
}
if cfg.IPC {
	services = append(services, makeRPCOverIpc(cfg.IPCPath, rpcServers, log, cfg.Metrics))
}
```

Reply:
> Websocket and grpc is being kept there too.

Both of those protocols use http, but I see your point. Let's leave the `ipcService` where it is for now.

> Maybe node/ipc.go > node/rpc.go is better solution? It would align with the way they are instantiated.

I like that idea. Let's leave it out of scope for this PR.